mubench.reviewsite/tests/StoreReviewTest.php

```diff
@@ -23,7 +23,7 @@ class StoreReviewTest extends DatabaseTestCase
             0 => [
                 'hit' => 'Yes',
                 'types' => [
-                    'missing/call'
+                    '1'
```

Comment:
Hm... I'm not really happy with this, because a change at a theoretically independent location can break this test... however, I plan to cleanup this types thing soon anyways, so let's leave it like this for now.